Scheduler/feature_scheduler/auxtel/feb_2025_fbs_spec_survey.py

```diff
@@ -0,0 +1,205 @@
+# This file is part of ts_config_ocs.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+from astropy import units
+from astropy.coordinates import Angle
+from lsst.ts.fbs.utils import Target
+from lsst.ts.fbs.utils.auxtel.surveys import (
+    generate_cwfs_survey,
+    generate_image_survey_from_target,
+    generate_spectroscopic_survey,
+    get_auxtel_targets,
+)
+from rubin_scheduler.scheduler.detailers import DitherDetailer
+from rubin_scheduler.scheduler.schedulers import CoreScheduler
+
+
+def get_scheduler():
+    """Construct feature based scheduler.
+
+    Returns
+    -------
+    nside : int
+        Healpix map resolution.
+    scheduler : Core_scheduler
+        Feature based scheduler.
+    """
+    # Information for all surveys
+    nside = 64
+    avoid_wind = False
+    if avoid_wind:
+        wind_speed_maximum = 13.0  # maximum direct wind in m/s
+    else:
+        wind_speed_maximum = 30
+
+    # Detailers - for now, just simple image or spectroscopy detailers.
+    image_detailers = [
+        # Dither up to half the FOV, per exposure.
+        DitherDetailer(max_dither=(7 / 2 / 60), per_night=False)
+    ]
+    spec_detailers = []
+
+    # Get target information - edit YAML file for updates
+    # Yaml file is in ts_fbs_utils :
+    # ts_fbs_utils/python/lsst/ts/fbs/utils/data/auxtel_targets.yaml
+    target_pointings = get_auxtel_targets()
+
+    # The targets below must be in the target_pointings
+    # But can be specified to be all or just a subset
+
+    # Imaging priority - high priority imaging, tier 1
+    imaging_priority_targets = ["Photo08000-1"]
+    # Imaging backup - low priority imaging, backup for everything - tier 3
+    imaging_backup_targets = ["Photo08000-1"]
```

Comment:
I think we can do away with the imaging_backup survey all together.

Reply:
I didn't check that the backup spectroscopy survey was really guaranteed to be available, all the time, so I figured having a backup that was going to provide targets to avoid idle time would be good. 
That said, this particular backup imaging survey may not be available all the time either and I was really thinking that a backup imaging style survey which was based on a greedy survey (so anywhere on the sky) but that just took a series of images with small dithers would be best. 
Maybe potential idle time is preferable though?

Reply:
Ok, I've removed it. 
It would've also been simple to leave the code there and just make the list of imaging_backup_targets = [], but .. eh, if you'll never use it, it's cleaner to remove.